pcsx2/windows/VCprojects/pcsx2.vcxproj

```diff
@@ -625,4 +629,4 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets" />
-</Project>
+</Project>
```

Comment:
Actually I try my best to not commit them. But My text editor often updates the end of file. I don't know what is the policy here.